R/repo-helper.R

```diff
@@ -0,0 +1,75 @@
+#' Create example repo for fledge demos
+#'
+#' @param open Whether to open the new project.
+#' @param pkg Package name.
+#' @param name Name for DESCRIPTION and git.
+#' @param email Email for DESCRIPTION and git.
+#' @param date String of time for DESCRIPTION and git.
+#' @param tempdir Directory within which to create the mock package folder.
+#' @param news If TRUE, create a NEWS.md file.
+#'
+#' @return
+#' @export
+#'
+create_fledge_mock_repo <- function(open = rlang::is_interactive(),
+                               pkg = "tea",
+                               name = "Kirill M\u00fcller",
```

Comment:
I wonder if we can get these from the Git config? Need to configure GitHub Actions though.

Reply:
Would you want the demo project to use a different name on say my computer? 
Also relevant https://github.com/r-lib/whoami

Reply:
Yeah, we could use the name and e-mail of the current user I think.

Reply:
@krlmlr this is used in the PR now.